### PR TITLE
Pretty

### PR DESCRIPTION
**Pull request by prosif**
Update to use new squish stuff in https://github.com/homegamesio/squish/pull/3 and make https://github.com/homegamesio/homegames/pull/34 work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,6 @@
   },
   "homepage": "https://github.com/homegamesio/homegames#readme",
   "dependencies": {
-    "squishjs": "^0.4.2"
```
Comment (YazeedLoonat) [inline package.json]: I think this version number needs to be up'ed

File: src/app.js [hunk 2]
```diff
@@ -63,27 +68,32 @@ const gameAssets = {};
 const imageCache = {};
 
 const canvas = document.getElementById("game");
-const ctx = canvas.getContext("2d", {alpha: false});
+const gameDiv = document.getElementById('homegames-main');
+const divColor = Colors.HG_YELLOW;//BLACK;
```
Comment (YazeedLoonat) [inline src/app.js]:
```suggestion
const divColor = Colors.COLORS.HG_YELLOW;//BLACK;
```

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]
I think after the issues after the rebase in the squish file, we would need to make updates here as well

**Comment by prosif** [comment]
Updated to use the version of squish currently in PR: https://github.com/homegamesio/squish/pull/3

Likely still busted in some way, but I need to update https://github.com/homegamesio/homegames/pull/34 to make _that_ work with squishjs 0.5.0 and then update this package to do the same. Basically the goal is "make master work"

**Comment by prosif** [comment]
Similar to https://github.com/homegamesio/homegames/pull/34, I'm going to merge this and do other cleanup/fixes in other PRs

**Pull request merged**